infrastructure/main.tf

```diff
@@ -127,6 +142,38 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
 }
 # endregion
 
+# region staging DB secrets
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES-USER" {
+  name         = "${var.component}-staging-POSTGRES-USER"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.user_name}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES-PASS" {
+  name         = "${var.component}-staging-POSTGRES-PASS"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.postgresql_password}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES_HOST" {
+  name         = "${var.component}-staging-POSTGRES-HOST"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.host_name}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES_PORT" {
+  name         = "${var.component}-staging-POSTGRES-PORT"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.postgresql_listen_port}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES_DATABASE" {
+  name         = "${var.component}--stagingPOSTGRES-DATABASE"
```

Comment:
typo - missing `-`

Reply:
> could use `count` for both `module` and `resource` declarations

Already discussed why we can't during refinement :)

Reply:
ok. certain sentences can slip through as kid is climbing on my head all the time during the meetings. that's his participation option :D

Reply:
problem with introducing count in upstream module is it might have breaking changes. That's was our first choice but platform too recommended doing it this way :)